### PR TITLE
create exam

### PR DESCRIPTION
**Pull request by namanh1997**
Screenshot from 2019-08-26 16-09-55

### CODE REVIEW

**Comment by namanh1997** [comment]
ready

**Comment by namanh1997** [comment]
ready

File: app/controllers/exams_controller.rb [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+  def index
+    @exams = @subject.exams.sort_by_name.page(params[:page])
+                     .per Settings.exams_per_page
+  end
+
+  def create
+    @exam = @subject.exams.build exam_params
+    @exam.number_question.times do
+      @exam.add_question(Question.where(subject_id: @subject.id).sample)
```
Comment (vanvtt-0952) [inline app/controllers/exams_controller.rb]: `.where(subject_id: @subject.id)` scope nè

File: app/helpers/exams_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module ExamsHelper
+end
```
Comment (vanvtt-0952) [inline app/helpers/exams_helper.rb]: Xóa hì

**Comment by namanh1997** [comment]
ready

**Pull request merged**